docs/kratos/social-signin/35_gitlab.mdx

```diff
@@ -4,8 +4,87 @@ title: Add GitLab as a social sign-in provider in Ory
 sidebar_label: GitLab
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # GitLab
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add GitLab as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the GitLab logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create a GitLab OAuth2 Application](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile).
+5. Paste the saved **Redirect URI** from Ory into the **Redirect URI** field in the GitLab form.
+6. Enable the relevant **Scopes** for the application, by ticking the relevant boxes in the GitLab form:
+    - `read_user`
+    - `openid`
+    - `profile`
+    - `email`
```

Comment:
Relevant x2, "Scopes" doesn't need to be formatted as you're not taking about UI elements, stick to the simple imperative form of "Tick the boxes to enable relevant scopes" or something along these lines.

Reply:
@fbolton Looks like you missed this comment.

Reply:
This does actually refer to the Scopes field in the GitLab form -- that's why it is bolded. And the "Scopes" in step 10 also refers to the a UI element - the Scopes field of the form in Ory.